### PR TITLE
PLFM-8145: fix virus scanner

### PR DESCRIPTION
**Pull request by xschildw**
This is last year's work where we downloaded the binaries from the website instead of using yumdownloader.
Looks OK from limited testing on dev stack.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,31 +15,17 @@ WORKDIR /opt/app
...
-RUN rpm2cpio clamav-lib*.rpm | cpio -idmv
-RUN rpm2cpio clamav-update*.rpm | cpio -idmv
-RUN rpm2cpio json-c*.rpm | cpio -idmv
-RUN rpm2cpio pcre*.rpm | cpio -idmv
-RUN rpm2cpio gnutls* | cpio -idmv
-RUN rpm2cpio nettle* | cpio -idmv
-RUN rpm2cpio lib* | cpio -idmv
-RUN rpm2cpio *.rpm | cpio -idmv
-RUN rpm2cpio libtasn1* | cpio -idmv
+RUN wget https://www.clamav.net/downloads/production/clamav-1.0.4.linux.x86_64.rpm
```
Comment (marcomarasca) [inline Dockerfile]: From https://www.clamav.net/downloads it seems the latest LTS is 1.0.6 (supported through 2025, see https://docs.clamav.net/faq/faq-eol.html), worth a try?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,9 @@
-FROM amazonlinux:2
+FROM public.ecr.aws/lambda/python:3.11
```
Comment (marcomarasca) [inline Dockerfile:1]: I'm not sure if we want to try the lastest 3.12 runtime, see https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

**Review by marcomarasca — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (xschildw) [reply Dockerfile]: That works on 1.06.

File: Dockerfile (hunk 2 above)
Comment (xschildw) [reply Dockerfile:1]: Tried and rolled back, the underlying OS changed to linux2023 so there's a bit of extra work to adapt to new env. Will open separate issue for that.

Comment (marcomarasca) [reply Dockerfile:1]: Sounds good!

**Review by marcomarasca — APPROVED** [review]
LGTM

**Review by marcomarasca — APPROVED** [review]

**Pull request merged**